pkgs/by-name/us/userborn/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  makeBinaryWrapper,
+  mkpasswd,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "userborn";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "nikstur";
```

Comment:
Since this could be a critical component of NixOS, it would be good if the bus factor eventually would get > 1 on the committers side of the repository.

Reply:
I agree. Always happy to accept contributions (incl. eventually moving the repo to another org)!